cve_bin_tool/cli.py

```diff
@@ -162,6 +162,13 @@ def main(argv=None):
         action="store",
         help="add a unique tag to differentiate between multiple intermediate reports",
     )
+    output_group.add_argument(
+        "-v",
+        "--verbose",
+        action="count",
+        default=0,
+        help="outputs affected versions in report",
+    )
```

Comment:
```suggestion
    output_group.add_argument(
        "--affected-versions",
        action="count",
        default=0,
        help="Lists versions of product affected by a given CVE (to faciliate upgrades)",
    )
```

I don't like "verbose" for this because it doesn't really explain what's being added to the report, and I think we can do more to explain to a user why they'd want to use this flag in the help text.

Reply:
And sorry, I know changing it here is going to require you to change it all over!  But I'd rather get it right in the PR stage than have to refactor it later.

Reply:
faciliTate

Reply:
Updated, thanks.  Not sure why my in-browser spell check missed that one!